sevntu-checks/src/main/java/com/github/sevntu/checkstyle/checks/coding/RedundantReturnCheck.java

```diff
@@ -338,9 +338,9 @@ private static DetailAST getRedundantReturnInBlock(DetailAST statementAst) {
      */
     private static DetailAST findRedundantReturnInCatch(DetailAST lastStatementInCatchBlockAst) {
         DetailAST redundantReturnAst = null;
-        DetailAST currentNodeAst = lastStatementInCatchBlockAst;
```

Comment:
The variable `currentNodeAst` removed as it is just a confusing alias for `lastStatementInCatchBlockAst`.